ArchiSteamFarm/ArchiSteamFarm.csproj

```diff
@@ -24,6 +24,7 @@
 		<PackageReference Include="SteamKit2" />
 		<PackageReference Include="System.Composition" />
 		<PackageReference Include="System.Security.Cryptography.ProtectedData" />
+		<PackageReference Include="Tmds.DBus.Protocol" />
```

Comment:
Would be nice to add a `Condition`; This will not be required for windows-only builds.